static/bidder-info/smarthub.yaml

```diff
@@ -13,7 +13,7 @@ capabilities:
       - video
       - native
 userSync:
-  # smarthub supports user syncing, but requires configuration by the host. contact this
+  # Attekmi supports user syncing, but requires configuration by the host. contact this
```

Comment:
Nitpick: perhaps change this to "Attekmi's Smarthub supports..."

Reply:
We provide the customer with updated setup instructions, so a name other than Attekmi would be misleading to the customer.